src/components/checkbox/checkbox.js

```diff
@@ -27,7 +27,12 @@ angular
  * @param {string=} ng-change Angular expression to be executed when input changes due to user interaction with the input element.
  * @param {boolean=} md-no-ink Use of attribute indicates use of ripple ink effects
  * @param {string=} aria-label Adds label to checkbox for accessibility.
- * Defaults to checkbox's text. If no default text is found, a warning will be logged.
+ *     Defaults to checkbox's text. If no default text is found, a warning will be logged.
+ * @param {expression=} md-indeterminant This determines when the checkbox
+ *     should be rendered as 'indeterminant'.
+ *     If a truthy expression or no value is passed in the checkbox renders in the md-indeterminant state.
+ *     If falsy expression is passed in it just looks like a normal unchecked checkbox.
+ *     To use the other md-indeterminant-* attributes this attribute must be present.
```

Comment:
In addition to that, I thought `md-indeterminat-*` attribute have been dropped? So maybe remove that line.